ansible/templates/zapd_block_height.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+dest_email=alerts@zap.me
+remote_block=`curl -s https://testnode1.wavesnodes.com/blocks/height | jq '.["height"]'` 
```

Comment:
remote url will be different for mainnet, make this come from a ansible variable